### ISSUE TITLE
Values don't seem to be available in FormData

### ISSUE THREAD

**Issue by lysdexic-audio** [janosh/svelte-multiselect#151]
Hoping to clarify. In the following I can access the formdata via `event.target` as normal for the input component, but not the value from the Multiselect:

```
(2) [ "martianflag" ,"" ]
(2) [ "name" ,"myname" ]
```

What is the intended method for `martianname` property to be assigned the value from the Multiselect component?

Working repro at this [REPL here](https://svelte.dev/repl/da8a01d4c0a8491f991bf7779a92a820?version=3.52.0)

```svelte
<script>
  import MultiSelect from 'svelte-multiselect'
  import ColorSlot from './ColorSlot.svelte'
  import { colors } from './options'
	let val, selectedopts;
	
	async function handleSubmit(event) {
		 const formData = new FormData(event.target);
		 console.log(...formData)
	}
</script>

<h2 align="center">Svelte MultiSelect</h2>

<form on:submit|preventDefault={handleSubmit}>
	<label for="colors">
		<strong>Which colors would you pick for the Martian flag?</strong>
	</label>
	<MultiSelect
		id="colors"
		options={colors}
		placeholder="Pick some colors..."
		allowUserOptions="append"
		maxSelect={1}
		bind:value={val}
		bind:selected={selectedopts}
		name="martianflag"
		required
	>
		<ColorSlot let:idx {idx} let:option {option} slot="selected" />
		<ColorSlot let:idx {idx} let:option {option} slot="option" />
	</MultiSelect>
	<div/>
	<input name="name" placeholder="name" required />
	<div/>
	<button>
		Submit
	</button>
</form>
```

**Comment by lysdexic-audio** [comment]
ah - I'm guessing that this isn't supported right now:

> "Applied to the <input> element. Sets the key of this field in a submitted form data object. Not useful at the moment since the value is stored in Svelte state, not on the <input> node.

This makes creating forms with other components alongside Multiselect a little clunky. Perhaps consider this issue an FR.

**Comment by janosh** [comment]
@lysdexic-audio Thanks for bringing this up. Been wanting to make this component more form friendly for a while. Always good to have a trigger for higher priority. I think passing the value of `selected` to form submission handlers should be a very simple fix.

**Comment by janosh** [comment]
Released as [v8.0.2](https://github.com/janosh/svelte-multiselect/releases/tag/v8.0.2).
